### ISSUE TITLE
Protect Screenshot not working on android

### ISSUE THREAD

**Issue by ernestoA92** [prongbang/screen_protector#10]
Protect Screenshot is not working on android. I've tested it on the latest versions and is not working although it used to work on early versions.

**Comment by mesahach** [comment]
yeah, mine too it`s not working on my Android and Emulator either.

**Comment by mdynia1** [comment]
Using this reply https://github.com/flutter/flutter/issues/40961#issuecomment-618724022  disables screenshots on all android devices, but there is no possibility to select specific screens
